packages/relic/test/headers/new/special/accept_encoding_header_test.dart

```diff
@@ -0,0 +1,262 @@
+import 'dart:io';
+import 'package:test/test.dart';
+import 'package:relic/src/headers/headers.dart';
+import 'package:relic/src/relic_server.dart';
+import '../../headers_test_utils.dart';
+
+void main() {
```

Comment:
Question: Do we give a reason in the reject message why the parsing failed? If so, maybe "strict" should be the one we test all header variations in. Then, we just validate that "null" is set if we fail.

That way, we can validate that the validation fails for the expected reason. For example, an invalid quality value.